cmake/onnxruntime_java.cmake

```diff
@@ -54,7 +54,7 @@ file(GLOB onnxruntime4j_native_src
 add_library(onnxruntime4j_jni SHARED ${onnxruntime4j_native_src} ${onnxruntime4j_generated})
 onnxruntime_add_include_to_target(onnxruntime4j_jni onnxruntime_session)
 target_include_directories(onnxruntime4j_jni PRIVATE ${REPO_ROOT}/include ${REPO_ROOT}/java/src/main/native)
-target_link_libraries(onnxruntime4j_jni PUBLIC ${JNI_LIBRARIES} onnxruntime onnxruntime4j_generated)
+target_link_libraries(onnxruntime4j_jni PUBLIC onnxruntime onnxruntime4j_generated)
```

Comment:
Ok for Linux but not Windows(I guess).

Reply:
Just tested it on Windows (as well as Linux and MacOS previously), and it worked fine :)
(Unrelated but hit the same issue as this one: https://github.com/microsoft/onnxruntime/issues/2892#issuecomment-579701427)

Reply:
As long there isn't upcalls from the native code to the JVM, that's fine.

Reply:
There are upcalls from the native binding into the JVM, it creates several objects to contain the results of computation and calls various static methods to translate between enum values. But because the JVM is already loaded it seems to be fine.